Content/posts/jwtkit-v5.md

```diff
@@ -0,0 +1,182 @@
+---
+date: 2024-02-22 16:00
+description: We've released a new version of JWTKit, and it's a big one!
+tags: jwt, security, vapor
+author: Paul; Tim
+authorImageURL: /author-images/paul.jpg
+--- 
+# JWTKit is no longer Boring!
+
+## Swift 6 is on the Horizon
+
+If you follow the Swift forums carefully you might have noticed [the announcement](https://forums.swift.org/t/progress-toward-the-swift-6-language-mode/68315/33) that Swift 5.10 will be the last release before Swift 6. This provides Vapor a timeline for a future Vapor 5 release and we can start planning as to what that will look like.
+
+A large part of that will be updating and migrating all of our packages to use modern Swift features and paradigms like `Sendable`, actors and making use of new APIs. The first package to be updated is JWTKit, which has been in the works for a while now.
```

Comment:
We've already been updating Vapor 4 stuff to be `Sendable`-correct - I would say something like "... to take advantage of modern Swift language features and design patterns, foremost among which is structured Swift Concurrency."